dev/com.ibm.ws.cdi.ejb/bnd.bnd

```diff
@@ -0,0 +1,83 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: com.ibm.ws.cdi.ejb
+Bundle-SymbolicName: com.ibm.ws.cdi.ejb
+Bundle-Description: CDI Common EJB, version ${bVersion}
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
+
+javac.source: 1.7
+javac.target: 1.7
+
+-dsannotations: com.ibm.ws.cdi.ejb.liberty.EjbEndpointServiceImpl,\
+  com.ibm.ws.cdi.ejb.impl.EjbSecurityContextStore,\
+  com.ibm.ws.cdi.ejb.liberty.JCDIHelperFactoryImpl
+
+Service-Component: \
+ com.ibm.ws.cdi.ejb.deserializationClassProvider;\
+    implementation:=com.ibm.ws.serialization.DeserializationClassProvider;\
+    provide:=com.ibm.ws.serialization.DeserializationClassProvider;\
+    properties:="service.vendor=IBM,\
+      packages=com.ibm.ws.cdi.ejb.impl\
+              |com.ibm.ws.cdi.ejb.liberty"
+  
+Private-Package: \
+  com.ibm.ws.cdi.ejb.*
+
+Import-Package: \
+                javax.decorator;version="[1.1,3)",\
+                javax.enterprise.context;version="[1.1,3)",\
+                javax.enterprise.context.spi;version="[1.1,3)",\
+                javax.enterprise.inject;version="[1.1,3)",\
+                javax.enterprise.inject.spi;version="[1.1,3)",\
+                com.ibm.ws.cdi.internal.interfaces, \
+                org.jboss.weld;version="[2.4,4)",\
+                org.jboss.weld.bean;version="[2.4,4)",\
+                org.jboss.weld.construction.api;version="[2.4,4)",\
+                !org.jboss.weld.ejb,\
```

Comment:
@Emily-Jiang why do I have to do this?!

Reply:
The reason is that it generates based on the bundle in the buildpath and then worked out the version range. The version range upper bound will be 3 not 4 based on the versioning scheme.